lib/casmutils/xtal/site.cxx

```diff
@@ -38,6 +38,8 @@ Eigen::Vector3d Site::frac(const Lattice& ref_lattice) const
     return ref_lattice.column_vector_matrix().inverse() * this->cart();
 }
 
+Site operator*(const sym::CartOp& sym_op, const Site& site) { return Site{sym_op * Coordinate(site), site.label()}; }
```

Comment:
This seems like a weird way of getting the coordinate. Is there a reason to not do `site.cart()`?

Reply:
I'm just using the * operator I wrote for coordinate. "site.cart()" returns the eigen vector stuff. I didn't want to re-implement the apply sym op again here.

Reply:
I understand now. Definitely good to reuse the implementation. I'd say you should take it a step further, and have `operator*` for `Eigen::Vector3d`, and use that for the `Coordinate` and `Site`. I think we're going to move away from a `Coordinate` class altogether in the future.

Reply:
Done